src/node_crypto.cc

```diff
@@ -3530,6 +3535,9 @@ bool Hash::HashInit(const char* hash_type) {
     return false;
   EVP_MD_CTX_init(&mdctx_);
   EVP_DigestInit_ex(&mdctx_, md_, nullptr);
+  if (0 != ERR_peek_error()) {
+    return false;
+  }
```

Comment:
@stefanmb Is there a reason you use `ERR_peek_error()` here instead of looking at the return value of `EVP_DigestInit_ex()`?

See https://github.com/nodejs/node/issues/4221 for context.